extensions/kubeapi/rbac/create.go

```diff
@@ -113,3 +113,25 @@ func CreateGlobalRoleBinding(client *rancher.Client, globalRoleBinding *v3.Globa
 
 	return newGlobalRoleBinding, nil
 }
+
+// CreateProjectRoleTemplateBinding is a helper function that uses the dynamic client to create a project role template binding.
+func CreateProjectRoleTemplateBinding(client *rancher.Client, prtb *v3.ProjectRoleTemplateBinding) (*v3.ProjectRoleTemplateBinding, error) {
+	dynamicClient, err := client.GetDownStreamClusterClient(LocalCluster)
+	if err != nil {
+		return nil, err
+	}
+
+	ProjectRoleTemplateBindingResource := dynamicClient.Resource(ProjectRoleTemplateBindingGroupVersionResource).Namespace(prtb.Namespace)
```

Comment:
Nit; this namespaced interface can be private 
```suggestion
	projectRoleTemplateBindingResource := dynamicClient.Resource(ProjectRoleTemplateBindingGroupVersionResource).Namespace(prtb.Namespace)
```